src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -401,15 +412,18 @@ case class Event(timestamp: Int,
                  method: String,
                  `object`: String,
                  value: Option[String],
-                 extra: Option[Map[String, String]]) {
+                 extra: Option[Map[String, Any]]) {
 
-  def getField(field: String): String = field match {
+  def getField(field: String): JValue = field match {
```

Comment:
Sorry, one more comment bc apparently I missed this change -- what's the reasoning re: the decision to make this return a JValue?

Reply:
This is related to extra changing to `Map[String, Any]`. We were previously relying on implicit conversions from String to JValue when using this function. That doesn't work generically if the return type here is `Any`, so changing the output type to `JValue` lets the implicit conversion happen within this function for most of the strings, and we explicitly convert where necessary.